src/components/hooks/useMeeting.js

```diff
@@ -48,13 +48,10 @@ export default function useMeeting(meetingID) {
 
       throw error;
     };
-    const onComplete = () => {
-      setMeeting(emptyMeeting);
-    };
 
     const subscription = meetingsAdapter
       .getMeeting(meetingID)
```

Comment:
let's only call this if the meetingID is set, otherwise we would have to implement this "protection" in each adapter (the json adapter is ok with a falsy meeting id, but the sdk adapter currently isn't). You can use the same approach as for a falsy `meetingDestination` in the `useMeetingDestination` hook